src/modules/roc_fec/reader.cpp

```diff
@@ -585,6 +534,10 @@ bool Reader::validate_incoming_repair_packet_(const packet::PacketPtr& pp) {
         }
     }
 
+    if (fec.payload.size() == 0) {
```

Comment:
Could we move this check into `check_payload_size_` ?

Reply:
Why? This is a part of packet validation.

In validate_incoming_source_packet_() and validate_incoming_repair_packet_() we drop invalid packets.

In check_payload_size_(), check_source_block_size_(), and check_repair_block_size_() we check whether the block or packet size update may be applied.

We probably can rename them somehow.

Reply:
I'll rename `check_*` to `can_update_*`.

Reply:
Renamed.